### PR TITLE
CLOGS Beta integration

### PR DESCRIPTION
**Pull request by parc-jason**
# Description
Send data to clogs, use an auto intercept.
Hardcoded the url for logstash for Beta.
Intercept all stdin/out and deliver to logstash.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-23

File: app/app.js [hunk 1]
```diff
@@ -39,6 +40,7 @@ log.debug('Config', utils.prettyStringify(config));
 
 // Skip if running tests
 if (process.env.NODE_ENV !== 'test') {
+  clogsSvc.hook();
```
Comment (jujaga) [inline app/app.js]: Do we also want this to not be mounted if we're in 'dev' mode?

File: app/src/components/clogsSvc.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const intercept = require('intercept-stdout');
+const isString = require('lodash.isstring');
+
+let unhook_intercept;
+let msgs = [];
+const clogsSvc = {
+
+  hook: () => {
+    if (!unhook_intercept) {
+      unhook_intercept = intercept(function (text) {
```
Comment (jujaga) [inline app/src/components/clogsSvc.js]: probably can be declared as an anonymous function instead
```
unhook_intercept = intercept(text => {
```

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-23

**Pull request merged**